File: services/acacia/src/routes/issues.ts
import { Router } from 'express';
import {
  getAllIssues,
  getIssueById,
  createIssue,
  updateIssue,
  deleteIssue,
} from '#acacia/controllers/issues.ts';

const router = Router();

router.get('/', getAllIssues);
router.get('/:id', getIssueById);
router.post('/', createIssue);
router.put('/:id', updateIssue);
router.delete('/:id', deleteIssue);

export default router;

File: services/acacia-go/internal/routes/issues.go
package routes

import (
	"acacia/internal/controllers"

	"github.com/go-chi/chi/v5"
)

func IssuesRoutes(controller *controllers.IssuesController) chi.Router {
	r := chi.NewRouter()

	r.Get("/", controller.GetAllIssues)
	r.Get("/{id}", controller.GetIssueByID)
	r.Post("/", controller.CreateIssue)
	r.Put("/{id}", controller.UpdateIssue)
	r.Delete("/{id}", controller.DeleteIssue)

	return r
}